tests/unittest/conftest.py

```diff
@@ -60,6 +60,15 @@ def _request_generator(method, path, query_string=None, headers=None, body=None)
 class AntennaTestClient(TestClient):
     """Test client to ease testing with Antenna API"""
 
+    @classmethod
+    def build_config(cls, new_config=None):
+        """Build ConfigManager using environment and overrides."""
+        new_config = new_config or {}
+        config_manager = ConfigManager(
+            environments=[ConfigDictEnv(new_config), ConfigOSEnv(),]
+        )
+        return config_manager
```

Comment:
This pulls from the environment and allows tests to override settings. Previously, it didn't pull from the environment at all.